teachertool/src/components/styling/CatalogOverlay.module.scss

```diff
@@ -111,6 +111,20 @@
                     }
                 }
             }
+        }
+
+                /* Styles for iPads (portrait mode) */
+        @media (min-width: 768px) and (max-width: 1024px) and (orientation: portrait) {
```

Comment:
where did these numbers come from? is this something that couldn't be done using the breakpoints in `react-common-breakpoints.less`?

Reply:
Teacher tool defines a set of breakpoints that mirrors react-common's, with a few added. Please use them: https://github.com/microsoft/pxt/blob/1e22ec7a07a5979d06b050bff407741a6e987c08/teachertool/src/global.scss#L144

Teacher tool doesn't have orientation breaks defined. Does react-common have these available? If not that would something useful to add (either in react-common or here in teacher tool). Something like:

```css
.landscape-only {
    display: initial;
}
.portrait-only {
    display: initial;
}
@media (orentation: landscape) {
    .portrait-only {
        display: none !important;
    }
}
@media (orientation: portrait) {
    .landscape-only {
        display: none !important;
    }
}
```

Reply:
@kimprice in case it got missed... I made a suggestion for integrating the orientation breakpoints.